components/automate-ui/src/app/pages/job-add/job-add.component.html

```diff
@@ -47,15 +47,15 @@
         <ng-container [ngSwitch]="step$ | async">
           <chef-job-nodes-form
             *ngSwitchCase="Step['add-nodes']"
-            [form]="form.controls['nodesGroup']">
+            [form]="formControl('nodesGroup')">
```

Comment:
I don't yet what the better way is - but when I test this out using a simple console log, we get a never-ending running function.  Similar to why we try to avoid using a function in an *ngFor.

Reply:
I imagine this is similar in the above files that also use a function in the template

Reply:
Excellent--glad you noticed the function-in-the-template faux pas! I was having a problem with loading scan jobs so had not been able to test this yet. In any case, I have pushed up a fix to replace the function calls with a _pure pipe_ (the other approach to resolving that as compared to my recent PR #3810 ).